_data/roles.yml

```diff
@@ -79,6 +79,25 @@
       orcid: 0000-0002-7356-1779
       status: support
       start: "0000-00-00"
+- name: OBO Slack Community Manager
+  open: false
+  commitment: 1-3 hours per month
+  requirements:
+    - love for communication
+    - passion for community building
+    - community mindset
+  description: You are responsible for managing the OBO Slack community. This role includes issues like welcoming new members, deleting old ones, sending invites and directing people to the right channels.
+  responsibilities:
+    - Adding people to the OBO slack space on request, using the 'invite members' feature
+    - Welcoming new members once per week in the #general channel
```

Comment:
put this line in quotes since # gets parsed as a comment otherwise

Reply:
```suggestion
    - "Welcoming new members once per week in the #general channel"
```